server/src/models/Medication.js

```diff
@@ -2,11 +2,14 @@ import mongoose from 'mongoose';
 
 const Schema = mongoose.Schema;
 const MedicationSchema = Schema({
+  key: String,
   drugName: String,
   quantity: Number,
   dosage: Number,
   units: String,
-  comments: String
+  comments: String,
+  lastUpdated: Number,
+  needToUpload: Boolean
```

Comment:
needToUpload is only relevant on local side

Reply:
I already made this change on my pull request